spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarAdministrationTests.java

```diff
@@ -52,6 +52,7 @@
  */
 @ExtendWith(SpringExtension.class)
 @ContextConfiguration
+@SuppressWarnings("JUnitMalformedDeclaration")
```

Comment:
This prevents the warnings from IDEA on commit saying that test methods should declare method parameters (which **IS** valid w/ junit5).